src/pages/Homepage.js

```diff
@@ -32,28 +35,61 @@ function Homepage() {
       <div className="w-full flex flex-col gap-24 md:flex-row  md:justify-center md:items-center md:min-h-[80vh] md:gap-0">
         <div className="hidden md:block">
           <button
```

Comment:
- This button is slightly closer to the cards than its counterpart, in the future if possible please make them the same distance.